drivers/pinmux/stm32/pinmux_stm32wbx.h

```diff
@@ -90,6 +90,11 @@
 #define STM32WBX_PINMUX_FUNC_PA7_TMR17_CH1 \
 	(STM32_PINMUX_ALT_FUNC_14 | STM32_PUSHPULL_NOPULL)
 
+/* USB */
+#define STM32WBX_PINMUX_FUNC_PA11_USB_DM STM32_PUSHPULL_NOPULL
+#define STM32WBX_PINMUX_FUNC_PA12_USB_DP STM32_PUSHPULL_NOPULL
```

Comment:
According to ST documents, it should be `(STM32_PINMUX_ALT_FUNC_10 | STM32_PUSHPULL_NOPULL)`

![image](https://user-images.githubusercontent.com/25011557/81102989-b066d780-8f10-11ea-91cb-8eae31142b35.png)

USB seems to be available only in `STM32WBx5`, not the value line `STM32WBx0`. So summarizing this code should be:

```c
#if CONFIG_SOC_STM32WB55XX
#define STM32WBX_PINMUX_FUNC_PA11_USB_DM \
        (STM32_PINMUX_ALT_FUNC_10 | STM32_PUSHPULL_NOPULL)
#define STM32WBX_PINMUX_FUNC_PA12_USB_DP \
        (STM32_PINMUX_ALT_FUNC_10 | STM32_PUSHPULL_NOPULL)
#endif
```

Reply:
@erwango do you know if pinmuxes for STM32 families is published in a text format? It would be nice to script the generation of pinmux headers.

Reply:
@gmarull Thanks! I see that now in the datasheet. Pushed the change.

Reply:
I think checkpatch is complaining because my code snippet contains spaces and not tabs :-)

Reply:
That's what I get for C&P'ing code..  

Pushed a fix with proper whitespace.

Reply:
> @erwango do you know if pinmuxes for STM32 families are published in a text format? It would be nice to script the generation of pinmux headers.

You can get them from CubeMx in some internal database in .xml format (in ~/STM32CubeMX/db/mcu).

Reply:
@erwango Got them, great discovery. Do you think that a script part of hal_stm32 would be useful? I can work on that to evaluate the feasibility.